app/view_models/auction_list_item.rb

```diff
@@ -64,10 +76,10 @@ def item_footer_partial
   end
 
   def winning_bid_partial
-    if auction.bids.any?
```

Comment:
Let's push the conditionals into this method rather than using two partials here